docs/tutorials/cloud-foundry.md

```diff
@@ -110,7 +120,9 @@ Now that we have a running CF app we can transform it using Move2Kube. We will b
 
     * For this tutorial, we have copied these files into the source directory already and renamed them as [*cfapps.yaml*](https://github.com/konveyor/move2kube-demos/blob/main/samples/cloud-foundry/cfapps.yaml) and [*cluster.yaml*](https://github.com/konveyor/move2kube-demos/blob/main/samples/cloud-foundry/cluster.yaml).
 
-2. Then we create a *plan* on how to transform your app to run on Kubernetes. In the *plan* phase, it is going to combine the runtime artifacts with source artifacts and going to come up with a *plan* for us.
+    * It is recommended to run the *collect* phase described in this step before running the *plan* phase. But, it is necessary to have the source artifacts of the CF application to run the *plan* phase.
```

Comment:
Instead of "recommended", do you want to say something like, "If runtime metadata such as env variables which are not in the manifest, but in the deployed app needs to be used in Kubernetes,...", So that our users know what we use the runtime info for

Reply:
Yes sure. Thank you.

Reply:
Done.